packages/shared/friends/sagas.ts

```diff
@@ -220,8 +220,10 @@ function* configureMatrixClient(action: SetMatrixClient) {
       if (userId) {
         // When it's a friend and is not added to catalog
         // unity needs to know this information to show that the user has connected
-        if (isFriend(store.getState(), userId) && !isAddedToCatalog(store.getState(), userId)) {
+        if (!isAddedToCatalog(store.getState(), userId)) {
           await ensureFriendProfile(userId)
+        }
+        if (isFriend(store.getState(), userId)) {
```

Comment:
I'm wondering about keeping this block the same as before instead. Two questions related to this:
1- What's the value of sending the profile to renderer if it's not a friend (at least for the logic here)?
2- Is it idempotent for renderer to `AddFriends` a given user id?

Reply:
1- Yes, you're right. It doesn't have any. So, instead, I changed the code to: 
```
if (isFriend(store.getState(), userId)) {
      if (!isAddedToCatalog(store.getState(), userId)) {
        await ensureFriendProfile(userId)
      }
      getUnityInstance().AddFriends({
        friends: [userId],
        totalFriends: getTotalFriends(store.getState())
      })
}
```
Because there is a scenario where we already sent the profile but not the friendship info.

2- Yes.

Reply:
Great, thank you!